os/fs/procfs/fs_procfs.c

```diff
@@ -210,6 +210,7 @@ const struct mountpt_operations procfs_operations = {
 	NULL,						/* sync */
 	procfs_dup,					/* dup */
 	NULL,						/* fstat */
+	NULL,
```

Comment:
Mention funcptr variable in comment section as reference.
Like /* truncate */

Reply:
Ok done, Please check